sources/packages/backend/apps/api/src/app.system-access.module.ts

```diff
@@ -78,6 +81,7 @@ import { WorkflowClientService, SequenceControlService } from "@sims/services";
     SFASIntegrationSystemAccessController,
     ECertIntegrationSystemAccessController,
     FedRestrictionsIntegrationSystemAccessController,
+    IERIntegrationSystemAccessController,
```

Comment:
Have we decided to use a controller temporarily to be able to test this? 

I remember in the grooming we spoke about a way in nest through which we can directly test the service.